charts/longhorn/latest/questions.yml

```diff
@@ -1,3 +1,4 @@
+rancher_max_version: 2.6.2
```

Comment:
We should allow Longhorn as this is one of the actively maintained charts by SUSE/Rancher.

Reply:
alright. I'll remove the rancher_max_version from longhorn.